snuba/migrations/snuba_migrations/transactions/0011_transactions_add_span_op_breakdowns.py

```diff
@@ -0,0 +1,64 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, Float, Nested, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    """
+    Adds the span_op_breakdowns nested column
+    """
+
+    blocking = False
+
+    def forwards_local(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.TRANSACTIONS,
+                table_name="transactions_local",
+                column=Column(
+                    "span_op_breakdowns",
+                    Nested(
+                        [
+                            ("key", String(Modifiers(low_cardinality=True))),
+                            ("value", Float(64)),
+                        ]
+                    ),
+                ),
+                after="measurements",
```

Comment:
This won't work. 
the `measurements` column does not exist after the DDL operation that creates it.
After such ddl you have `measurements.key` and `measurements.value` columns.
So this should be 
`after="measurements.value"`

Reply:
@fpacifici Good catch. I updated to `after="measurements.value"`. 

Interestingly, the `span_op_breakdowns` columns are added after the `measurements` columns in my local Clickhouse instance.